aztec-up/test/counter_contract.sh

```diff
@@ -56,7 +56,7 @@ cd counter_contract
 sed -i 's|\.\./\.\./\.\./\.\./|/home/ubuntu/aztec-packages/noir-projects/|g' Nargo.toml
 
 # Compile and codegen.
-aztec-nargo compile
+aztec-nargo compile --pedantic-solving
```

Comment:
```suggestion
aztec-nargo compile
```

aztec-nargo is injecting this as well which is causing CI to fail.